.github/workflows/_unit_tests.yaml

```diff
@@ -0,0 +1,36 @@
+name: tests base
```

Comment:
BTW: that feels a bit inaccurate. This is plainly the unit test workflow, isn't it?